File: 4oy1 Muhriddin Sobirjonov/8masala.py
import  os
os.system("cls")

a, b, c, d = map (int, input("4 ta son kiriting  ").split())
juft = 0
if  a % 2 == 0:
    juft += 1
if  b % 2 == 0:
    juft += 1
if  c % 2 == 0:
    juft += 1
if  d % 2 == 0:
    juft += 1

if juft >= 2:
    print(a, b, c, d, sep="  ")
else:
    print(0)

File: 3oy6 Muhriddin Sobirjonov/8masala.c
#include    <stdio.h>

int sum = 1;
int  FOKTARIAL(int a, int son){
    sum *= a;
    if(a < son){
        return FOKTARIAL(a + 1, son);
    }
    printf("sonning foktariali ==> %d", sum);
}

int main(){
    system("cls");
    int son;
    printf("sonni kiriting:  ");
    scanf("%d", &son);
    FOKTARIAL(1, son);

    return 0;
}